src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --main-color-font: #000;
+  --main-color-font-is-hover: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: var(--main-color-font);
+  text-decoration: none;
+  padding: 23px 0;
```

Comment:
its better not to set paddings here, just use height: 100% or `aling-items: center` in .header

Reply:
in this case i have a problem and it doesnt work when i add class .header for header and add aling-items: center or height: 100%. so i left padding without change